docs/en/concepts-and-designs/OAP-Cluster-Management.md

```diff
@@ -0,0 +1,39 @@
+# Observability Analysis Platform Cluster Management
+OAP(Observability Analysis Platform) server is a distributed system, services need to find each 
+other. i.e. a web service needs to find query service, level 1 aggregate service need to find 
+level 2 aggregate service. Cluster management is just a client-side implementation. It must work 
+together with a distributed coordination server, (i.e. Zookeeper, Consul, Kubernetes.) unless 
```

Comment:
Should be `distributed coordination service`.  Service is Zookeeper says for itself. Ref https://zookeeper.apache.org/